projects/uitdatabank/docs/entry-api/school-events.md

```diff
@@ -0,0 +1,356 @@
+
+> If you’re planning to create school events in UiTdatabank with the Entry API, please contact <content.cultuurkuur@publiq.be> in advance so that we can guide you through this process.
+
+## Types
+
+We distinguish 3 different types of school events:
+
+1. **School performances** are events of which both the date and the location is known in advance.
+2. **Guided tours** are events that have no specific date, but they do have a location.
+3. **Bookable events** are events that have no specific date and location (or the date and location is not known in advance). Both the date and the location are determined in mutual agreement between the organizer or artist and the consumer (school).
+
+### School performances
+
+School performances are events of which both the date and the location is known in advance. For example, a theater performance aimed at a toddler of 3-4 years old in hetpaleis on 14/05/2023, 14:30 - 16:00 PM.
+
+* ✅ date is known in advance
+* ✅ location is known in advance
+
+Since school performances both have a date and a location, you can create them in a very similar way to regular events:
+
+* you can use a calendarType of your preference (`single`, `multiple` or `periodic`)
+* for the location you must use the URL of an existing place as `location.@id` in the `POST /events` request of the event
+
+### Guided tours
+
+Guided tours are events that have no specific date (or the date is not known in advance), but they do have a location. For example, a guided tour in the Royal Museum of Fine Arts Antwerp.
+
+* ❌ date is not known in advance
+* ✅ location is known in advance
+
+In order to create a guided school tour you must use:
+
+* calendarType `permanent`
+* the URL of an existing place as `location.@id` in the `POST /events` request of the event
+
+### Bookable events
+
+Bookable events are events that have no specific date and location (or the date and location is not known in advance). Both the date and the location are determined in mutual agreement between the organizer or artist and the consumer (school).
+
+For example, as a school you can book Stijn Meuris for a school performance at your school or local cultural centre, on a date that suits both the artist and the school.
+
+* ❌ date is not known in advance
+* ❌ location is not known in advance
+
+In order to create a bookable school event you must use
+
+* calendarType `permanent`
+* the URL of the dummy place as `location.@id` in the `POST /events` request of the bookable events
+
+**URL of the dummy place for bookable events**:
+
+* Test environment: `https://io-test.uitdatabank.be/place/3b92c85b-a923-4895-85f5-ed056dae11e2`
+* Production environment: `https://io.uitdatabank.be/place/c3f9278e-228b-4199-8f9a-b9716a17e58f`
+
+## Creating school events
+
+For the creation of school events several extra requirements apply:
+
+1. The `audienceType` must be set to `education`
```

Comment:
I think this is technically not required when using a "location in consultation with the school" `@id` (aka the "dummy location" mentioned right now) because UDB3 then automatically sets the audienceType to "education" and also prevents it from being changed, but on the other hand it does not harm either and for school performances and guided tours it needs to be set explicitly of course. So you could mention that it's not required in that one case of bookable events, but that might also make it more confusing. Just mentioning it so you can decide with all the possible info.

Reply:
Good remark! But I'm going to omit that nuance to avoid confusion